src/common/aggregate/aggregator.js

```diff
@@ -10,6 +10,10 @@ export class Aggregator extends SharedContext {
   constructor (parent) {
     super(parent)
     this.aggregatedData = {}
+
+    /* Unfortunately, there's no other good medium through which SR can signal it's ready to other features that depends on it.
```

Comment:
The event emitter?

Reply:
It switches to unbuffered when everything drains, and there's no guarantee that `registerHandler` from another feature agg on an event emitted by SR agg will win the race, I think.